azure/azure-explanation/security-overview.rst

```diff
@@ -0,0 +1,109 @@
+Ubuntu Security on Azure
+========================
+
+Ubuntu security features
+------------------------
+
+Ubuntu on Azure inherits and benefits from all of the security features available to Ubuntu Server. Learn more about
+Ubuntu Server security features and security-related topics with `Introduction to Security <https://documentation.ubuntu.com/server/explanation/intro-to/security/>`_.
+
+Azure security features
+-----------------------
+
+By default, all Ubuntu X64 Hyper-V Gen2 images on Azure support `Trusted Launch for Azure Virtual Machines <https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch>`_.
+Ubuntu also produces images which support `Azure Confidential Virtual Machines <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview>`_.
+
+See :doc:`confidential-computing`.
+
+Choosing the right image
+------------------------
+
+Ubuntu offers a range of image types on Azure to meet security requirements. The security features and capabilities
+of each image can be further enhanced with `Ubuntu Pro`_.
+
+Server
+~~~~~~
+
+Optimised for general server use cases, Ubuntu Server is ideal for users seeking a stable and secure OS foundation to
+support server workloads.
+
+Minimal
+~~~~~~~
+
+By providing a greatly reduced package set to minimise the security cross-section of the image, Ubuntu Minimal requires
+fewer security updates over time.
```

Comment:
I would like to put promote minimal a bit more here. I would like to see Server only used for pet servers and Minimal use for "Cattle" but more generally for everything that doesn't involve a lot of user interaction. Minimal should also be preferred when building golden images. They are a good base for anything automated.

Reply:
As we briefly discussed already, as this is the `Security Overview` document the product descriptions should be limited in scope to the security features and not creep into broader use cases and applications.